app/build.gradle

```diff
@@ -127,47 +127,48 @@ androidExtensions {
 
 ext {
     // App dependency versions
-    supportVersion = '27.1.1'
-    playServicesVersion = '15.0.1'
-    daggerVersion = '2.14.1'
-    retrofitVersion = '2.3.0'
-    okHttpVersion = '3.9.1'
+    supportVersion = '1.0.0-beta01'
```

Comment:
We will probably want to drop this and move to individually versioning the support dependencies.